tensorflow/compiler/mlir/hlo/tests/Dialect/mhlo/ops.mlir

```diff
@@ -965,7 +965,7 @@ func.func @rng_bit_generator(%arg0: tensor<2xui64>) -> (tensor<2xui64>, tensor<1
   %4 = mhlo.constant dense<[10, 12]> : tensor<2xui64>
   %0 = mhlo.constant dense<[10, 12]> : tensor<2xi32>
   %1 = mhlo.constant dense<3> : tensor<i32>
-  %2, %3 = "mhlo.rng_bit_generator"(%4) {rng_algorithm = 0 : i32} : (tensor<2xui64>) -> (tensor<2xui64>, tensor<10x12xui32>)
+  %2, %3 = "mhlo.rng_bit_generator"(%4) {rng_algorithm = #mhlo<"rng_algorithm DEFAULT">} : (tensor<2xui64>) -> (tensor<2xui64>, tensor<10x12xui32>)
   func.return %2, %3 : tensor<2xui64>, tensor<10x12xui32>
```

Comment:
What error do you get with `mhlo.rng_algorithm<DEFAULT>`? (No quotes)

Reply:
error message:
```
error: unexpected error: expected string literal data in dialect symbol
  %2, %3 = "mhlo.rng_bit_generator"(%4) {rng_algorithm = #mhlo<DEFAULT>} : (tensor<2xui64>) -> (tensor<2xui64>, tensor<10x12xui32>)
```
and this is quotes version:
```
error: unexpected error: unknown mhlo attribute
  %2, %3 = "mhlo.rng_bit_generator"(%4) {rng_algorithm = #mhlo<"DEFAULT">} : (tensor<2xui64>) -> (tensor<3xui64>, tensor<10x12xui32>)
```

Reply:
I refer to the definition of MLIR Core.Maybe we need assemblyFormat.
```
def HLO_RngAlgorithmAttr : EnumAttr<HLO_Dialect, HLO_RNG_ALGORITHM, "rng_algorithm"> {
  let assemblyFormat = "`<` $value `>`";
}
```
then use ```{rng_algorithm = #mhlo.rng_algorithm<DEFAULT>}``` it work well.